frontend/src/components/HomePage/Card.jsx

```diff
@@ -6,8 +6,8 @@ const Card = ({ bill, billCampaignMappings, allCampaigns }) => {
   let categories = [];
   if (billCampaignMappings && allCampaigns) {
     categories = billCampaignMappings
-      .map((abrv) => allCampaigns[abrv].short_name)
-      .filter((v) => !!v);
+      .map((abrv) => allCampaigns[abrv])
+      .filter(Boolean);
```

Comment:
I think this filter was checking if `short_name` existed, which this does not do.

Reply:
Good point. I fixed it (with a fallback on long_name just in case)